extras/cloudsql-multicluster/README.md

```diff
@@ -174,4 +218,26 @@ Example output, from an EU-based GCE instance:
 Cluster: boa-2, Pod: frontend-74675b56f-2ln5w, Zone: europe-west3-a
 ```
 
+> **Note:** You can create a GCE instance in a European region and try the same
```

Comment:
Praise: Thanks for adding this tip about creating a virtual machine. So convenient! 
It was fun hitting the `europe-west3-a` cluster!
```Cluster: boa-2, Pod: frontend-6888bc6676-26pkp, Zone: europe-west3-a```